src/event.rs

```diff
@@ -353,8 +353,10 @@ pub fn process_event(ctx: &mut Context, event: &winit::event::Event<()>) {
                 };
                 ctx.keyboard_context.set_key(*keycode, pressed);
             }
-            winit_event::WindowEvent::ScaleFactorChanged { .. } => {
-                // Nope.
+            winit_event::WindowEvent::ScaleFactorChanged { new_inner_size, .. } => {
+                // actively set the new_inner_size to be the current inner size
+                // to stop winit from resizing our window
+                **new_inner_size = ctx.gfx_context.window.window().inner_size();
```

Comment:
How does this not change new_inner_size for you?
I just don't get it...

Reply:
any logs to capture on my side?

Reply:
something like?

![image](https://user-images.githubusercontent.com/449224/120806205-26f16100-c579-11eb-8366-8da3f9abd114.png)

![image](https://user-images.githubusercontent.com/449224/120806180-1f31bc80-c579-11eb-85fa-49f3d12f53a3.png)